### PR TITLE
[TASK] Unify confvals for config section

### PR DESCRIPTION
**Pull request by linawolf**
Replace aspect table with confval where not yet happened.

Use unified names

We should have a look at the default values in a follow up. Some are missing, some are integers for type boolean.

Releases: main, 12.4

### CODE REVIEW

**Comment by garvinhicking** [comment]
Phew, this is hard to review, I could offer a vote of confidence only 😜

File: Documentation/Setup/Config/Index.rst [hunk 1]
```diff
@@ -437,91 +397,78 @@ compressJs
...
 
-   Example
-         .. code-block:: typoscript
-            :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+    config.concatenateJs = 1
 
-            config.concatenateJs = 1
+    page = PAGE
+    page.includeJSFooter {
+    test = fileadmin/user_upload/test.js
```
Comment (brotkrueml) [inline Documentation/Setup/Config/Index.rst:466]: Please indent correctly.

File: Documentation/Setup/Config/Index.rst [hunk 2]
```diff
@@ -1568,26 +1329,22 @@ noPageTitle
...
+~~~~~~~
 
-   Example
-         .. code-block:: typoscript
-            :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
 
-            pageRendererTemplateFile = fileadmin/test_pagerender.html
+    pageRendererTemplateFile = EXT:my_extension/Ressources/Public/Templates/TestPagerender.html
```
Comment (brotkrueml) [inline Documentation/Setup/Config/Index.rst:1347]:
```suggestion
    pageRendererTemplateFile = EXT:my_extension/Ressources/Private/Templates/TestPagerender.html
```

This should not be exposed to the outside as this is used internally by PageRenderer.

File: Documentation/Setup/Config/Index.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/Setup/Config/Index.rst]: Done

**Review by brotkrueml — APPROVED** [review]
Reviewed it and it is fine for me, although haven't tested it. So, go ahead.

**Pull request merged**